source/invoker.js

```diff
@@ -27,6 +27,16 @@ import toString from './toString';
  *      sliceFrom(6, 'abcdefghijklm'); //=> 'ghijklm'
  *      const sliceFrom6 = R.invoker(2, 'slice')(6);
  *      sliceFrom6(8, 'abcdefghijklm'); //=> 'gh'
+ *
+ *      const dog = {
+ *        speak: async () => 'Woof!'
+ *      };
+ *      const person = {
+ *        speak: async () => 'Hi!'
+ *      };
```

Comment:
Is there a strong reason for giving two separate examples of the same behavior here?  I think either `dog` or `person` would be fine.

Reply:
The reasoning was to show that `invoker` can be used on either piece of data. I agree it's not necessary, will remove.

Reply:
updated